other/materials_designer/create_interface_with_no_strain_matching.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
<p>Enable scaling of the film slab atomic coordinates to match the substrate lattice (preserve coordinates in crystal units).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>